### PR TITLE
Update developer site for invoice api

### PR DESCRIPTION
**Pull request by mbrameshkumar**
This PR addresses the following

- Convert the invoice api documentation style to use RAML v1.0
- Structure changes due to migration
- Update the supported customer numbers
- Sample response

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/yAG2vPAV](https://trello.com/c/yAG2vPAV)

File: api/invoice/api.raml [hunk 1]
```diff
@@ -1,66 +1,183 @@
...
+        example: 25.01.2020
+    responses:
+      200:
+        displayName: Successful response
+        description: |
+          The shipment is invoiced based on invoice source and issuer. Here are a list of supported invoice issuers that can be returned.
+
+          | Invoice issuer | Description |
+          |:-------|:--------|
+          | `BRING_CARGO_AS` | `Bring Cargo AS` |
```
Comment (anirbansingharoy) [inline api/invoice/api.raml]: I liked the idea here to document this but I am wondering if its needed. These organization we don't specifically show anywhere . Its used in the react app to show invoice source.

File: api/invoice/api.raml [hunk 2]
```diff
@@ -1,66 +1,183 @@
...
+          | `BRING_EIENDOM_HAUGESUND_AS` | `Bring Eiendom Haugesund AS` |
+          | `POSTEN_NORGE_AS` | `Posten Norge AS` |
+          | `POSTEN_NORGE_AS_FOR_CARGO_SEA` | `Posten Norge AS` |
+          | `OTHER` | `Other` |
+
+          Here are a list of invoice types that can be returned.
+
+          | Invoice Source |
+          |:-------|
+          | `INVOICE` |
```
Comment (anirbansingharoy) [inline api/invoice/api.raml]: I think the values we show here is "Invoice" , "Credit Note"

Comment (anirbansingharoy) [reply api/invoice/api.raml]: I never seen any invoices with "Deposit" and "Debit Note" . Let me check with BKL if they seen any data for these types else we can remove it all together

File: api/invoice/api.raml (hunk 1 above)
Comment (mbrameshkumar) [reply api/invoice/api.raml]: I also feel the same. we can remove it if it is used technically. Earlier I didn't know where it is used.

File: api/invoice/api.raml (hunk 2 above)
Comment (mbrameshkumar) [reply api/invoice/api.raml]: Let me know the outcome. I will make the changes

**Pull request merged**